src/omero_cli_render.py

```diff
@@ -68,13 +71,13 @@
 
     # Omitted fields will keep their current values, omitted channel-indices
     # will be turned off.
-    bin/omero render edit --copy Screen:1 <YAML or JSON file>
-    # Optimised for bulk-rendering, edits the first image and copies the
+    bin/omero render set --copy Screen:1 <YAML or JSON file>
+    # Optimised for bulk-rendering, sets the first image and copies the
     # rendering settings to the rest. Note using this flag may have different
     # results from not using it if the images had different settings to begin
```

Comment:
This needs a heavy reformulation please. I guess this is the main strenght of the whole plugin, but must be explained in simple sentences. I still do not understand what is it supposed to do.